sdk/database/dbplugin/databasemiddleware.go

```diff
@@ -314,7 +315,7 @@ func (mw *DatabaseErrorSanitizerMiddleware) sanitize(err error) error {
 		return nil
 	}
 	if errwrap.ContainsType(err, new(url.Error)) {
-		return errors.New("unable to parse connection url")
+		return fmt.Errorf("unable to parse connection url, err=%w", err)
```

Comment:
Will update with better error handling

Reply:
Does `err` contain the connection URL and can we safely return that in this case?